terraform-tests/redis_test.go

```diff
@@ -39,7 +39,7 @@ var _ = Describe("Redis", Label("redis-terraform"), Ordered, func() {
 
 	Context("with Default values", func() {
 		BeforeAll(func() {
-			plan = ShowPlan(terraformProvisionDir, buildVars(defaultVars, map[string]any{}))
+			plan = ShowPlan(terraformProvisionDir, buildVars(defaultVars, map[string]any{"node_type": "cache.t3.medium"}))
```

Comment:
thoughts: it seems reasonable to add `"node_type": "cache.t3.medium"` to the `defaultVars`

Reply:
That's a good point.
I can try removing `cache_size` from `defaultVars` and create a different testing that `cache_size` still works as expected.

However, by doing so, the specific test for `cache_size` won't be able to use `defaultVars`. So if your concern is maintainabiltiy and reducing code, this might not be a good solution.

I would have a try, push a new commit and you can take a look to decide if it looks better.
I believe keeping `cache_size: 2` in defaultVars doesn't make any sense anymore.

Reply:
Changed in commit: https://github.com/cloudfoundry/csb-brokerpak-aws/pull/774/commits/e1b98c701d764d72ef155faf667adf7868ceb771